src/app/components/viewer/viewer.component.ts

```diff
@@ -190,7 +190,6 @@ export class AppViewerComponent implements OnInit, OnDestroy {
   }
 
   onViewerVisibilityChanged() {
-    this.store.dispatch(new ReloadDocumentListAction());
```

Comment:
causes
```
ERROR in src/app/components/viewer/viewer.component.ts:33:3 - error TS6133: 'ReloadDocumentListAction' is declared but its value is never read.
33   ReloadDocumentListAction,
     ~~~~~~~~~~~~~~~~~~~~~~~~
```

you probably need to cleanup the imports

Reply:
Updated